bin/crystal

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env sh
+#!/bin/sh
```

Comment:
For example, [`checkbashisms`](https://linux.die.net/man/1/checkbashisms) won't recognize the script as an `sh` script if you keep the `/usr/bin/env sh` shebang.